### DISCUSSION TITLE
请问binlog load里，mysql表和doris表字段不同时要怎么创建同步任务？

### DISCUSSION THREAD

**Discussion by llzzll** [apache/doris#10149] [A - General / Q&A]
假设原表为 test.test ，字段为a,b,c,d,e
doris表为 test1，字段为a,c,e
请问SYNC JOB里的字段映射要怎么创建，文档中对这块的描述过于模糊

**Comment by hf200012** [comment]
建议你使用stream load ，json数据格式导入
